lib/Bazaarvoice/bvseosdk.php

```diff
@@ -785,7 +785,7 @@ private function _replaceTokens($content) {
       // contents.
       htmlspecialchars(
         $this->config['base_url'] . $page_url_query_prefix,
-        ENT_QUOTES | ENT_HTML5,

```

Comment:
@hellboy1975 rather than just removing this, would it be better to wrap it in a version check or similar?